tests/utils/suite_writer/slash_run_result.py

```diff
@@ -111,7 +111,7 @@ def __getitem__(self, args):
 
     def __contains__(self, args):
         return self._normalize_key(args) in self._events_by_args
-        return self.has_event(*args)
+        # return self.has_event(*args)
```

Comment:
@ayalash well has_event just calls `__contains__` again, so isn't it a Maximum recursion depth exceeded exception?